### ISSUE TITLE
Pending still spinning after the transaction is sent and received

### ISSUE THREAD

**Issue by mlequo** [asgardex/asgard-exchange#112]
Top right corner "pending" is spinning long after the transaction is sent and received

![Screen Shot 2021-02-09 at 11 10 45 AM](https://user-images.githubusercontent.com/78820962/107417625-a474fb00-6aca-11eb-8d50-9073909bb3a0.png)

**Comment by stuartkuentzel** [comment]
Hey @mlequo :wave: We kind of recently updated how we're handling pending transactions in this PR https://github.com/asgardex/asgard-exchange/commit/0b091d407191c0a15c031d44ea8e37d060c15747

Basically, issues were coming up when a transaction had gone through on the main chain, say BTC, but THORChain hadn't picked it up yet. So BTC says a success, but it's still pending on THORChain, causing confusing. If you notice this again, feel free to reopen or raise another issue.
